### PR TITLE
build: add docker compose for development

### PR DESCRIPTION
**Pull request by macintoshpie**
# background
current docker-compose is designed for prod. Development setup is configured with local psql, python env, etc.

# changes
Add a docker compose stack for development
- use django dev server for live reload (mounted code dir into container)
- DEBUG=True
- log emails to console

# test

run `docker-compose down && docker-compose -f docker-compose.yml -f docker-compose.dev.yml up`

### CODE REVIEW

File: bsyncviewer/settings/docker_dev.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+from bsyncviewer.settings.docker import *  # noqa: F401, F403
+
+DEBUG = True
+
```
Comment (aviveiros11) [inline bsyncviewer/settings/docker_dev.py:16]: Im not sure if this is just something on my end but I had to add `ALLOWED_HOSTS = ['0.0.0.0']` to this file.

**Review by aviveiros11 — APPROVED** [review]
just one question but otherwise works for me after adding the docker volumes.

File: docker-compose.dev.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# Usage:
+#   docker-compose -f docker-compose.yml -f docker-compose.dev.yml up
+version: '3'
+services:
+  web:
+    command: python3 manage.py runserver 0.0.0.0:80
+    environment:
+      - DJANGO_SETTINGS_MODULE=bsyncviewer.settings.docker_dev
+    volumes:
+      - ./bsyncviewer:/srv/selection-tool/bsyncviewer
```
Comment (nllong) [inline docker-compose.dev.yml:11]: I'm sure this was quite some time ago, but it is great to see that you can use relative paths now in docker compose!

**Review by nllong — APPROVED** [review]

**Comment by macintoshpie** [comment]
@nllong can you merge this or upgrade my permissions? I don't have permission to merge currently
Screen Shot 2021-05-05 at 10 13 14 AM

**Comment by nllong** [comment]
how 'bout i do both @macintoshpie -- you should be able to merge approved PRs now!

**Pull request merged**